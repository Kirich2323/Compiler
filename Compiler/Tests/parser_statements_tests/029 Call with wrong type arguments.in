function f(a, b : integer) : integer;
begin
end;
var
    a : integer = -10;
begin
    f(10.0, a);
end.