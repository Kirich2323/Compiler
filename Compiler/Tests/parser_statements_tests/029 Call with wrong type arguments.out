(7 ; 6): Type error: Got "float". Expected: "integer".